static/src/index.js

```diff
@@ -1,5 +1,7 @@
 import React from "react";
```

Comment:
Hi Jon,

I'm not quite sure what you mean by this. My understanding is that since this is not a purely React app so I can't really just follow the code structure given on that site. From what I've read on create-react-app.dev, I haven't found any material on integrating React with a framework like Flask; I only came across something on integration with an API backend. Upon Googling 'flask react boilerplate', one of the Github repos that came up was the the one that joshgav used to base his Flask-React-Postgres app on. 

Could you elaborate on what you mean here? I'm probably misunderstanding you or missing some pieces of info here. 

Thanks!

-Lily

Reply:
Create react app is kind of a standard these days for creating react apps, it handles a lot of the stuff that I'm seeing issues with above, like css/js dependency managements, etc.  It sets you up with a good base to work from.  You can put this on the backlog to do later if you'd like.